### PR TITLE
Bug/remove deprecated component gamemode button

### PR DESCRIPTION
**Pull request by OniWarrior**
In this ticket, Rhiannon Stanford and I removed a deprecated component, GamemodeButton, and all of the imports and Routes associated with this component.

Reference trello card for detailed description of what was removed.

Trello Card: https://trello.com/c/CssjNAzA

### CODE REVIEW

**Review by artofmayhem — APPROVED** [review]
effective and clean removal of the old button utilizing the anonymous trig function. Unsure why that was iterated in such a manner to begin with. But very nice fix and adjustment to a cleaner instantiation. 

Nice code cleanup in index.js

File: src/components/pages/Gamemode/Gamemode.js [hunk 1]
```diff
@@ -113,10 +113,6 @@ const Gamemode = ({ ...props }) => {
               </Row>
             </div>
           )}
-
-      {props.child.gamemode.sp && props.child.gamemode.mode === 'single' && (
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:117]: Nice, boolean was not needed

File: src/index.js [hunk 2]
```diff
@@ -113,7 +113,6 @@ function App() {
       <Switch>
         <Route exact path="/gamemode" component={Gamemode} />
         <Route path="/gameplay" component={GamePlayMain} />
-        <Route path="/gamemode/single" component={GamemodeButton} />
```
Comment (just-kato) [inline src/index.js:116]: Nice work

File: src/components/pages/Gamemode/index.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
 // Gamemode Feature
 export { default as Gamemode } from './Gamemode';
-export { default as GamemodeButton } from './GamemodeButton';
```
Comment (just-kato) [inline src/components/pages/Gamemode/index.js:3]: Great Job with scrubbing this unnecessary export

File: src/components/pages/Gamemode/Gamemode.js [hunk 4]
```diff
@@ -6,7 +6,6 @@ import { connect } from 'react-redux';
 
 import { useHistory } from 'react-router-dom';
 import { Link, Route } from 'react-router-dom';
-import GamemodeButton from './GamemodeButton';
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:9]: Again nice work, the less redundancy the better!

File: src/components/pages/Gamemode/Gamemode.js [hunk 5]
```diff
@@ -72,16 +72,7 @@ const Gamemode = ({ ...props }) => {
       setsP(true);
     }
   };
-  const trig = () => {
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:76]: Great Job!

File: src/index.js [hunk 6]
```diff
@@ -32,7 +32,7 @@ import {
 import { AddChild } from './components/pages/AddChild';
 import { ChildDashboard } from './components/pages/ChildDashboard';
 import { DrawingSub } from './components/pages/DrawingSub';
-import { Gamemode, GamemodeButton } from './components/pages/Gamemode';
```
Comment (just-kato) [inline src/index.js:35]: Nice

File: src/components/pages/Gamemode/Gamemode.js [hunk 7]
```diff
@@ -96,9 +96,7 @@ const Gamemode = ({ ...props }) => {
 
               <Row>
                 <Col className="adventure-passport" xs={16} sm={24}>
-                  {!sP && props.child.gamemode.mode === 'select' && (
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:109]: Good eye with removing that trig call

File: src/components/pages/Gamemode/Gamemode.js [hunk 8]
```diff
@@ -96,7 +99,7 @@ const Gamemode = ({ ...props }) => {
 
               <Row>
                 <Col className="adventure-passport" xs={16} sm={24}>
-                  <Button>Single Player</Button>
+                  <Button onClick={startSinglePlayerMode}>Single Player</Button>
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:102]: looks much better

**Review by just-kato — APPROVED** [review]
Great Job Stephen!

**Pull request merged**